src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -409,13 +413,39 @@ class DeviceAttestationVerifier
     void EnableCdTestKeySupport(bool enabled) { mEnableCdTestKeySupport = enabled; }
     bool IsCdTestKeySupported() const { return mEnableCdTestKeySupport; }
 
+    void SetRevocationDelegate(DeviceAttestationRevocationDelegate * delegate) { mRevocationDelegate = delegate; }
+
 protected:
     CHIP_ERROR ValidateAttestationSignature(const Crypto::P256PublicKey & pubkey, const ByteSpan & attestationElements,
                                             const ByteSpan & attestationChallenge, const Crypto::P256ECDSASignature & signature);
 
     // Default to support the "development" test key for legacy purposes (since the DefaultDACVerifier)
     // always supported development keys.
     bool mEnableCdTestKeySupport = true;
+
+    DeviceAttestationRevocationDelegate * mRevocationDelegate = nullptr;
```

Comment:
Why is this on the interface here?  Shouldn't it be on whatever class uses DeviceAttestationRevocationDelegate to implement CheckForRevokedDACChain?

I guess the issue is that some consumer (e.g. chip-tool?) just does GetDefaultDACVerifier() and then wants to set this delegate on it. But this part doesn't make sense for other verifier impls....

What should probably happen is that `GetDefaultDACVerifier` grows either an optional argument or a two-argument overload or something.

Reply:
Even I didn't want it here, but wasn't sure where to put it. Thanks for the suggestion. Moved to `DefaultDACVerifier`. and adjusted the bits in chip-tool to set it up accordingly. PTAL